backend/api/src/test/kotlin/software/shonk/adapters/incoming/ShorkInterpreterControllerV1IT.kt

```diff
@@ -640,4 +648,42 @@ class ShorkInterpreterControllerV1IT : AbstractControllerTest() {
             assertTrue(response.errors.isNotEmpty())
         }
     }
+
+    @Nested
+    inner class GetLobbySettings {
+        @Test
+        fun `test get lobby settings for an existing lobby`() = runTest {
+            client.post("/api/v1/lobby") {
+                contentType(ContentType.Application.Json)
+                setBody("{\"playerName\":\"playerA\"}")
+            }
+            val result = client.get("/api/v1/lobby/0/settings")
```

Comment:
Here you act like the lobby ID returned will definetly be 0. Sequential lobby IDs starting from 0 are not defined anywhere. Please use the lobby ID returned in the response instead, this way it will not fail if the lobby ID allocation is changed in the future.